env-dist

```diff
@@ -21,7 +21,7 @@ OIDC_RP_CLIENT_SECRET_ENCODED = True
 OIDC_OP_DOMAIN = ''
 OIDC_OP_STORE_ACCESS_TOKEN = True
 # Assuming user remo with password remo and db name db_1
-DATABASE_URL = 'mysql://remo:remo@127.0.0.1:3306/db_1'
+DATABASE_URL = 'mysql://remo:remo@db:3306/remo'
```

Comment:
I think this file should be modified as the need to individual. Its just a sample env file